lib/Manager.php

```diff
@@ -228,7 +229,8 @@ public function createRoomObject(array $row): Room {
 			(string) $row['object_id'],
 			(int) $row['breakout_room_mode'],
 			(int) $row['breakout_room_status'],
-			(int) $row['call_recording']
+			(int) $row['call_recording'],
+			(int) $row['can_mention_everyone']
```

Comment:
```suggestion
			(int) $row['can_mention_everyone'],
```

Reply:
Did you know a easy way to put this at the linter?
Maybe we can open a tech debt issue about this do don't lost this subject